app/models/actor_attributes.rb

```diff
@@ -417,7 +417,12 @@ def sustainability_for_group(name, node_type, include_totals)
     end
     {
       name: name,
-      included_columns: [{name: node_type.humanize}] + risk_indicators.map{ |indicator| indicator.slice(:name, :unit) },
+      included_columns:
+          [{name: node_type.humanize}] +
+          risk_indicators.map {
+              |indicator| quant = Quant.find_by(name: indicator[:backend_name])
+            {name: indicator.key?(:name) ? indicator[:name] : quant.frontend_name , unit: quant.unit}
```

Comment:
AGROSATELITE_SOY_DEFOR_ has a slightly different label in the database, so for now we need to keep this fallback logic here. Maybe this is something to address on the new db structure